src/invidious.cr

```diff
@@ -345,6 +345,12 @@ Invidious::Routing.get "/preferences", Invidious::Routes::PreferencesRoute, :sho
 Invidious::Routing.post "/preferences", Invidious::Routes::PreferencesRoute, :update
 Invidious::Routing.get "/toggle_theme", Invidious::Routes::PreferencesRoute, :toggle_theme
 
+Invidious::Routing.get "/setup_2fa", Invidious::Routes::Accounts, :setup_2fa_page
+Invidious::Routing.post "/setup_2fa", Invidious::Routes::Accounts, :setup_2fa
+Invidious::Routing.get "/remove_2fa", Invidious::Routes::Accounts, :remove_2fa_page
+Invidious::Routing.post "/remove_2fa", Invidious::Routes::Accounts, :remove_2fa
+Invidious::Routing.post "/validate_2fa", Invidious::Routes::Accounts, :validate_2fa
```

Comment:
I think that it'd be a good idea to have everything under `/2fa/` (`/2fa/setup`, `/2fa/remove`, etc..).